src/Framework/Models/ValueObjects/Relationship.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Give\Framework\Models\ValueObjects;
+
+use Give\Framework\Support\ValueObjects\Enum;
+
+/**
+ * @method static ONE_TO_ONE();
+ * @method static ONE_TO_MANY();
+ */
+class Relationship extends Enum {
+    const ONE_TO_ONE = 'one-to-one';
+    const ONE_TO_MANY = 'one-to-many';
```

Comment:
Let's add:
- MANY_TO_MANY
- BELONGS_TO
- BELONGS_TO_MANY

Make sure to update the models as makes sense.

Reply:
Okay updated 77ada265

Do we want to stick with `ONE_TO_ONE`, `ONE_TO_MANY` or `HAS_ONE`, `HAS_MANY`?

Reply:
I personally think the `HAS` syntax is easier to comprehend so added 04d0efcb.  can use either or.

Reply:
Yeah, that's a good idea.

Reply:
Let's just remove the generic ones. They're not clear.

Reply:
all set db24f3a2